src/style.css

```diff
@@ -36,7 +36,7 @@
   font-weight: 600;
   box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
   text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
-  border: 1px solid #e1e7ed;
+  border: 1px solid blue;

```

Comment:
Remove this rule. I think, it can be reason to fail tests.

Reply:
I tried, it is not a reason